debug.log

```diff
@@ -0,0 +1,5 @@
+[0824/224008.645:ERROR:process_info.cc(359)] UncheckedAllocate
```

Comment:
Add debug.log to .gitignore. We don't need to commit this to the repository.

add something like `*.log` since we don't want to commit any logs. If sharing logs is a thing, we can set something up, but I don't think this is something that should be checked in.